programs/native/bpf_loader/src/lib.rs

```diff
@@ -29,6 +29,53 @@ fn dump_program(key: &Pubkey, prog: &[u8]) {
     }
 }
 
+#[allow(unused_variables)]
+pub fn helper_abort_verify(
+    unused1: u64,
+    unused2: u64,
+    unused3: u64,
+    unused4: u64,
+    unused5: u64,
+    ro_regions: &[MemoryRegion],
+    unused7: &[MemoryRegion],
+) -> Result<(()), Error> {
+    Err(Error::new(
+        ErrorKind::Other,
+        "Error: BPF program called abort()!",
+    ))
+}
+
+#[allow(unused_variables)]
+pub fn helper_abort(unused1: u64, unused2: u64, unused3: u64, unused4: u64, unused5: u64) -> u64 {
+    // Never called because its verify function always returns an error
+    0
+}
+
+#[allow(unused_variables)]
+pub fn helper_sol_panic_verify(
+    unused1: u64,
+    unused2: u64,
+    unused3: u64,
+    unused4: u64,
+    unused5: u64,
+    ro_regions: &[MemoryRegion],
+    unused7: &[MemoryRegion],
+) -> Result<(()), Error> {
+    Err(Error::new(ErrorKind::Other, "Error: BPF program Panic!"))
+}
+
+#[allow(unused_variables)]
+pub fn helper_sol_panic(
+    unused1: u64,
+    unused2: u64,
+    unused3: u64,
+    unused4: u64,
+    unused5: u64,
+) -> u64 {
+    // Never called because its verify function always returns an error
```

Comment:
it's

Reply:
I want the possessive

Reply:
got it. I read it as "it's a verify function"